xml/depl_maintenance.xml

```diff
@@ -419,14 +535,239 @@
       &compnode;s needs to be sufficient to evacuate the nodes one by one.
      </para>
     </listitem>
+    <listitem>
+     <para>
+      &o_img; needs to be set up with shared storage; that means the
+      <guimenu>Default Storage Store</guimenu> value in the &o_img; must
+      <emphasis>not</emphasis> be set to <literal>File</literal>.
```

Comment:
Got it! So is the following correct?

*If the Default Storage Store value in Glance is set to File, then Glance must be configured as a shared storage.*